<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C5MainWindow</class>
 <widget class="QMainWindow" name="C5MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C5MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTreeWidget" name="twDb">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnection"/>
   <addaction name="actionAppend_database"/>
   <addaction name="actionLogin"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnection">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/database.png</normaloff>:/database.png</iconset>
   </property>
   <property name="text">
    <string>Connection</string>
   </property>
   <property name="toolTip">
    <string>Connection settings</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/lock.png</normaloff>:/lock.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="toolTip">
    <string>Login</string>
   </property>
  </action>
  <action name="actionAppend_database">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/database_new.png</normaloff>:/database_new.png</iconset>
   </property>
   <property name="text">
    <string>Append database</string>
   </property>
   <property name="toolTip">
    <string>Append database</string>
   </property>
  </action>
  <action name="actionConfigure_connection">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/configure.png</normaloff>:/configure.png</iconset>
   </property>
   <property name="text">
    <string>Configure connection</string>
   </property>
   <property name="toolTip">
    <string>Configure connection</string>
   </property>
  </action>
  <action name="actionClose_application">
   <property name="text">
    <string>Close application</string>
   </property>
   <property name="toolTip">
    <string>Close application</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
